platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -127,16 +133,26 @@
 
     private LDAPConnectionFactory ldapConnectionFactory;
 
+    private ServiceReference serviceReference;
+
+    public SslLdapLoginModule() {
```

Comment:
Added it on Phil's recommendation to inject the EncryptionService instead of overriding the LoginModule's decrypt-password method. I didn't realize someone was already calling initialize. I'll delete the constructor and re-test.